doc/content/verification_and_validation/ver-1fd.md

```diff
@@ -25,12 +25,18 @@ In [!cite](ambrosek2008verification), the value of $k = 801$ W/m/K is provided,
 
 The comparison between TMAP8 predictions and the analytical solution is performed at depth $x = 5$ cm.
 These results are shown in [ver-1fd_comparison_convective_heating].
-They show great agreement between TMAP8 and the analytical solution with a root mean square percentage error of RMSPE = 0.04 %.
+They show great agreement between TMAP8 and the analytical solution with a root mean square percentage error of RMSPE = 0.29 %.
```

Comment:
How is the RMSPE increased? Was the number wrong in the first place?

Reply:
The value is different with the one in documentation after rerunning the python script. So I update it with latest value.